vm/os-windows.cpp

```diff
@@ -304,6 +304,61 @@ void handle_ctrl_c() {
   SetConsoleCtrlHandler(factor::ctrl_handler, TRUE);
 }
 
+static volatile bool stop_on_ctrl_break = false;
+static HANDLE ctrl_break_thread = nullptr;
+
```

Comment:
Ah I see. I never heard of nullptr before. But please use NULL for now and in later pr's we can tidy it up.